include/revng/Model/DynamicFunction.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "revng/Model/GeneratedDynamicFunction.h"
+
+/// Function defined in a dynamic library
+class model::DynamicFunction : public model::generated::DynamicFunction {
+public:
+  using generated::DynamicFunction::DynamicFunction;
+
+public:
+  Identifier name() const;
+
+public:
+  bool verify() const debug_function;
+  bool verify(bool Assert) const debug_function;
+  bool verify(VerifyHelper &VH) const;
+  void dump() const debug_function;
+};
+
+#include "revng/Model/GeneratedDynamicFunction_late.h"
+
+static_assert(validateTupleTree<model::DynamicFunction>(IsYamlizable));
```

Comment:
Can we push this in the generated code?
Should always be true.

Reply:
There already is such an assert in the generated code :) I just missed it from the cleanup, removed!